### ISSUE TITLE
Updating to i3status-rust-0.14.4-1 has broken my i3status-rust

### ISSUE THREAD

**Issue by epixinvites** [greshake/i3status-rust#1052]
I'm running on Arch Linux 5.10.16 and I recently updated i3status-rust. When I reloaded i3, it simply showed 
`Error: Could not parse JSON (parse error: integer overflow)` but when I run i3status-rust in the terminal it works perfectly fine.

**Comment by epixinvites** [comment]
After a couple of digging, I found out which block was causing the problem. Apparently the music block wasn't working.
```
#[[block]]
#block = "music"
#player = "ncspot"
#buttons = ["play", "next"]
#max_width = 18
#dynamic_width = true
#marquee_interval = 5
#marquee_speed = 0.8
#hide_when_empty = true
```
i commented it out and it worked fine

**Comment by ammgws** [comment]
Same as #1047, fixed in 0.14.5
